securitycenter/google/cloud/securitycenter_v1/gapic/security_center_client.py

```diff
@@ -307,6 +308,19 @@ def create_source(
         request = securitycenter_service_pb2.CreateSourceRequest(
             parent=parent, source=source
         )
+        if metadata is None:
+            metadata = []
+        metadata = list(metadata)
+        try:
+            routing_header = [("parent", parent)]
+        except AttributeError:
```

Comment:
I've complained before about this bit of codegen:  the `AttributeError` cannot be raised here (no attribute is accessed).